systems/configuration.nix

```
@@ -68,6 +68,8 @@
         PermitEmptyPasswords = "no";
         PermitRootLogin = "no";
         TcpKeepAlive = "no";
+        PermitTTY = "no";
+        X11Forwarding = false;
```

we might can leave X11For. to false, since it not needed usually. but then we still need to use lib.mkDefault
```suggestion
        X11Forwarding = lib.mkDefault false;
```